experiments/README.md

```diff
@@ -0,0 +1,15 @@
+# Installation of VerifAI and Scenic
+
+1. Clone this repository and [Scenic](https://github.com/BerkeleyLearnVerify/Scenic).
+2. Checkout the `kesav-v/multi-objective` branch of VerifAI and the `kesav-v/multi-objective` branch of Scenic.
```

Comment:
Update this (and tricky bit below about doing the installs in the right order) assuming we have a new release of Scenic which integrates your code. Then just cloning VerifAI and installing as usual should be enough.